src/internal/storage/fileset/postgres_cache_test.go

```diff
@@ -37,8 +37,8 @@ func TestPostgresCache(t *testing.T) {
 		valueProto := &TestCacheValue{FileSetId: id.HexString()}
 		data, err := proto.Marshal(valueProto)
 		require.NoError(t, err)
-		value := &types.Any{
-			TypeUrl: "/" + proto.MessageName(valueProto),
+		value := &anypb.Any{
+			TypeUrl: "/" + string(proto.MessageName(valueProto)),
```

Comment:
Just curious if this was auto-inserted by any tooling?  Or if proto.MessageName actually returns a type distinct from `string`

Reply:
Yeah that's weird.  It returns a string.

Reply:
Nevermind.  In gogo it was a string, in google it is a protoreflect.FullName, which is `type FullName string`, not `type FullName = string`.